finance/jobs/2020_githubresponder.rst

```diff
@@ -0,0 +1,68 @@
+GitHub First/Last Responder
+===========================
+
+The Astropy Project is looking to hire a Github first responder / last
+responder. The overarching goal of this position is to improve the
+first time contributor experience, reduce the technical barriers to
+contributions, reduce the number of open issues, and increase the
+number of merged pull requests. This is not intended to cover all
+issues and PRs opened in astropy, but rather to support and lighten
+the load on the rest of the volunteer community.  An ideal candidate
+would be familiar with the Astropy project ecosystem and capabilities,
+comfortable with its Github Development workflow, and has experience
```

Comment:
not sure how to put this into the ad, but this role requires advanced `git` knowledge and comfort in working with multi fork environments. Resolving stalled PRs basically always requires rebases, squashes, cherry-picks, sorting out messed up histories, while keeping contributor credits.